deepdrivemd/aggregation/stream/config.py

```diff
@@ -0,0 +1,16 @@
+from typing import Optional
+from deepdrivemd.config import AggregationTaskConfig
+from pathlib import Path
+
+class StreamAggregation(AggregationTaskConfig):
+    rmsd: bool = True
```

Comment:
Explanatory comments for each parameter would be helpful

Reply:
Done